internal/provider/resource_vault_cluster_test.go

```diff
@@ -307,6 +308,8 @@ func updateVaultPublicEndpointObservabilityDataAndMVU(t *testing.T, in *inputT)
 			resource.TestCheckResourceAttrSet(in.VaultClusterResourceName, "audit_log_config.0.datadog_api_key"),
 			resource.TestCheckResourceAttr(in.VaultClusterResourceName, "audit_log_config.0.datadog_region", "us1"),
 			resource.TestCheckResourceAttr(in.VaultClusterResourceName, "major_version_upgrade_config.0.upgrade_type", "MANUAL"),
+			resource.TestCheckResourceAttr(in.VaultClusterResourceName, "vault_plugin.0.plugin_name", "venafi-pki-backend"),
+			resource.TestCheckResourceAttr(in.VaultClusterResourceName, "vault_plugin.0.plugin_type", "SECRET"),
```

Comment:
so that test case isn't included yet because there is only 1 customer managed plugin to work with. I did some hacky things to test 2 of them. Once more plugins are added, we can update the tests.

hafsa had the same confusion, sorry if this wasn't clear in pr description

Reply:
Ohh gotcha. Suggestion: Could we maybe have update 3 use a config with no plugins at all and then assert `resource.TestCheckNoResourceAttr` so then acceptance tests run by anyone will test removal without needing the [extra setup](https://github.com/hashicorp/terraform-provider-hcp/pull/575#discussion_r1291595165) you did?